pkgs/applications/networking/protonvpn-gui/default.nix

```diff
@@ -14,6 +14,7 @@
   proton-vpn-connection,
   proton-vpn-killswitch,
   proton-vpn-killswitch-network-manager,
+  proton-vpn-local-agent,
```

Comment:
It's a bit weird to add the package after pvpn-gui has been updated.

Just init the package stand-alone before and add it in the update that requires it.

Reply:
Moved this commit before the protonvpn-gui package.

Reply:
I may not have been clear enough, what I meant was to init the package independently and only add it in the update commit. This way pvpn is always in a sensible buildable state.

Reply:
I understand. That being said, with the current commit tree structure, protonvpn-gui is always in a buildable state, no matter what commit is being selected. Just to confirm, I've selected the commit right before the package update and it built without any issues.

Reply:
It builds, sure, because the package is simply ignored but it just doesn't make sense for pvpn-gui 4.4.4 to have it in its build closure.

If someone were to look at what changes were done in order to make 4.6.0 work by looking at the git commits, they wouldn't see the local-agent being added unless they opened a browser and had a look at this PR.

It's clearer for the diff of 4.6.0 to be the one adding the package to pvpn-gui. That way it also reverts to a sensible state.

Reply:
Makes sense. So, just to confirm, should I create a create a clean PR for the local agent in here just to add it here for the update?